### PR TITLE
VR > Cost/Actions z-index

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/134145

🎥 https://www.loom.com/share/60fabf2f4ca14abe957e226427301279

### CODE REVIEW

File: variables/_z-index.pcss [hunk 1]
```diff
@@ -11,5 +11,6 @@
 	--z-index-scroller: 7;
 	--z-index-week-event-hover: 5;
 	--z-index-map-event-hover: 5;
+	--z-index-map-event-hover-actions: 6;
```
Comment (paulmskim) [inline variables/_z-index.pcss]: I wonder if we should spread the other z-indices out so we have spaces between 5, 6, and 7. If we have any z-index needed between any of these numbers, we'll need to move things around.

Comment (paulmskim) [reply variables/_z-index.pcss]: I mean maybe we should do it now rather than later.

Comment (juanfra) [reply variables/_z-index.pcss]: I though that at some point, but given that are related to other views I thought that by only making it higher that `event-hover` is more than enough. (And also smaller than the index for the scroller).

Comment (paulmskim) [reply variables/_z-index.pcss]: Sure, fair enough. Just something for us to think about next time we add z-indices in between other z-indices.

Comment (paulmskim) [reply variables/_z-index.pcss]: Also once we share this with Green Team we're going to have to really watch how they work with each other.

**Pull request merged**